assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -16,21 +16,27 @@ import { trackEvent as trackEventGA } from 'helpers/ga';
 import Bundle from './Bundle';
 import ContribAmounts from './ContribAmounts';
 import { changeContribType } from '../actions/bundlesLandingActions';
-
+import getSubsLinks from '../helpers/subscriptionsLinks';
 
 import type { Contrib, Amounts } from '../reducers/reducers';
+import type { SubsUrls } from '../helpers/subscriptionsLinks';
 
 
 // ----- Types ----- //
 
+// Disabling the linter here because it's just buggy...
+/* eslint-disable react/no-unused-prop-types */
+
 type PropTypes = {
   contribType: Contrib,
-  contribAmount: Amounts, // eslint-disable-line react/no-unused-prop-types
-  intCmp: string, // eslint-disable-line react/no-unused-prop-types
+  contribAmount: Amounts,
```

Comment:
Yeah, because it started throwing lint errors on all of them for whatever reason. It just can't handle props when they're passed out to other functions.